src/cow-react/modules/limitOrders/pure/Orders/OrdersTabs.tsx

```diff
@@ -1,6 +1,16 @@
 import styled from 'styled-components/macro'
-import { useCallback, useState } from 'react'
 import { Trans } from '@lingui/macro'
+import { Link } from 'react-router-dom'
+
+export const LIMIT_ORDERS_TAB_KEY = 'tab'
+
+export const buildLimitOrdersTabUrl = (pathname: string, search: string, tabId: string) => {
```

Comment:
Would it make sense to move this to a generic util, and pass just:

```
{
  pathName: string,
  search: string,
  update: [key: string]: string
}
```

So you can pass in the update 
```
{
  tab: tabId 
}
```

Reply:
It's a pretty local function, originally created only for `OrdersTabs` component